devicetypes/fibargroup/fibaro-flood-sensor-zw5.src/fibaro-flood-sensor-zw5.groovy

```diff
@@ -1,338 +1,346 @@
-    log.debug "ManufacturerSpecificReport"
-    log.debug "manufacturerId:   ${cmd.manufacturerId}"
-    log.debug "manufacturerName: ${cmd.manufacturerName}"
-    log.debug "productId:        ${cmd.productId}"
-    log.debug "productTypeId:    ${cmd.productTypeId}"
+	log.debug "ManufacturerSpecificReport"
+	log.debug "manufacturerId:	 ${cmd.manufacturerId}"
+	log.debug "manufacturerName: ${cmd.manufacturerName}"
+	log.debug "productId:		 ${cmd.productId}"
+	log.debug "productTypeId:	 ${cmd.productTypeId}"
 }
 
 def zwaveEvent(physicalgraph.zwave.commands.manufacturerspecificv2.DeviceSpecificReport cmd) {
-    log.debug "DeviceSpecificReport"
-    log.debug "deviceIdData:                ${cmd.deviceIdData}"
-    log.debug "deviceIdDataFormat:          ${cmd.deviceIdDataFormat}"
-    log.debug "deviceIdDataLengthIndicator: ${cmd.deviceIdDataLengthIndicator}"
-    log.debug "deviceIdType:                ${cmd.deviceIdType}"
+	log.debug "DeviceSpecificReport"
+	log.debug "deviceIdData:				${cmd.deviceIdData}"
+	log.debug "deviceIdDataFormat:			${cmd.deviceIdDataFormat}"
+	log.debug "deviceIdDataLengthIndicator: ${cmd.deviceIdDataLengthIndicator}"
+	log.debug "deviceIdType:				${cmd.deviceIdType}"
 }
 
 def zwaveEvent(physicalgraph.zwave.commands.versionv1.VersionReport cmd) {
-    log.debug "VersionReport"
-    log.debug "applicationVersion:      ${cmd.applicationVersion}"
-    log.debug "applicationSubVersion:   ${cmd.applicationSubVersion}"
-    log.debug "zWaveLibraryType:        ${cmd.zWaveLibraryType}"
-    log.debug "zWaveProtocolVersion:    ${cmd.zWaveProtocolVersion}"
-    log.debug "zWaveProtocolSubVersion: ${cmd.zWaveProtocolSubVersion}"
+	log.debug "VersionReport"
+	log.debug "applicationVersion:		${cmd.applicationVersion}"
+	log.debug "applicationSubVersion:	${cmd.applicationSubVersion}"
+	log.debug "zWaveLibraryType:		${cmd.zWaveLibraryType}"
+	log.debug "zWaveProtocolVersion:	${cmd.zWaveProtocolVersion}"
+	log.debug "zWaveProtocolSubVersion: ${cmd.zWaveProtocolSubVersion}"
 }
 
 def zwaveEvent(physicalgraph.zwave.commands.batteryv1.BatteryReport cmd) {
-    log.debug "BatteryReport"
-    log.debug "cmd: "+cmd
-    log.debug "location: "+location
-    def timeDate = location.timeZone ? new Date().format("yyyy MMM dd EEE h:mm:ss a", location.timeZone) : new Date().format("yyyy MMM dd EEE h:mm:ss")
-    //def timeDate = new Date().format("yyyy MMM dd EEE HH:mm:ss", location.timeZone)
-    sendEvent(name: "battery", value: cmd.batteryLevel)
-    sendEvent(name: "batteryStatus", value: "Battery: $cmd.batteryLevel%\n($timeDate)")
+	log.debug "BatteryReport"
+	log.debug "cmd: "+cmd
+	log.debug "location: "+location
+	def timeDate = location.timeZone ? new Date().format("yyyy MMM dd EEE h:mm:ss a", location.timeZone) : new Date().format("yyyy MMM dd EEE h:mm:ss")
+	def value = cmd.batteryLevel
+	def desc = "Curent battery level"
+	def isStateChange = false

```

Comment:
This variable should be left to the internal logic of `sendEvent` and `createEvent` unless it's the special case of low battery.